cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -12,10 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from argparse import ArgumentError
 from typing import DefaultDict, Dict, Sequence, TYPE_CHECKING, Optional
 
 import abc
 from collections import defaultdict
+import cirq
+
 
 from cirq import circuits, devices, ops, protocols
```

Comment:
```suggestion

from cirq import circuits, devices, ops, protocols, transformers
```

Reply:
But don't we need to import cirq for `@cirq.transformer` and `@cirq._compat.deprecated_class`? Or should I try to import the modules for these too?